react/__tests__/StoreLink.test.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import { screen, render } from '@vtex/test-tools/react'
+
+import StoreLink from '../StoreLink'
+
+test('StoreLink Component', () => {
+  render(<StoreLink href="/custom-link" label="View More" title="View More" />)
+
+  const link = screen.getByTitle('View More')
```

Comment:
Did you try to get the link by label?
```suggestion
test('StoreLink Component', () => {
  const { getByTitle } = render(<StoreLink href="/custom-link" label="View More" title="View More" />)

  const link = getByTitle('View More')
```

Remember to remove the screen from imports.

Reply:
@igorbrasileiro Updated.